src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -1545,6 +1547,56 @@ namespace ngraph
                 }
                 writer.block_end();
             }
+
+            template <>
+            void GPU_Emitter::EMITTER_DECL(ngraph::op::gpu::Lstm)
+            {
+                // auto lstm = static_cast<const ngraph::op::gpu::Lstm*>(node);
+
+                // auto& cudnn_emitter =
+                //     external_function->get_primitive_emitter()->get_cudnn_emitter();
+                // size_t lstm_index = cudnn_emitter->build_primitive(lstm);
+
+                // writer << "gpu::invoke_primitive(ctx, " << lstm_index << ", ";
+                // writer << "std::vector<void*>{" << args.front().get_name();
+                // for (size_t i = 1; i < args.size(); i++)
+                // {
+                //     writer << ", " << args[i].get_name();
+                // }
+                // writer << "}.data(), ";
+                // writer << "std::vector<void*>{" << out.front().get_name();
+                // for (size_t i = 1; i < out.size(); i++)
+                // {
+                //     writer << ", " << out[i].get_name();
+                // }
+                // writer << "}.data()";
+                // writer << ");\n";
+            }
+
+            template <>
+            void GPU_Emitter::EMITTER_DECL(ngraph::op::gpu::Rnn)
```

Comment:
As you already mentioned it's request newer version of CUDNN. we should add a version guard somewhere. 

Since currently  we don't support old CUDNN version, we can fall back to the normal ops for old CUDNN version.